src/modules/QuestsList/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { useEffect, useState } from 'react'
+import Grid from '../../ui/Grid/Grid'
+import {Link} from 'react-router-dom'
+import Button from '../../ui/Button/Button'
+import userRoutes from '../../routes/user'
+//this is a component where user can see available quests that can be accepted
+interface QuestList {
+  // id: number
+  // name: string 
+  // points: number
+  // description: string
+  // encounters: number
+  // type: string
+  match: object
+}
+
+
+type QuestProps = QuestList
+
+const QuestList: React.FC<QuestProps> = (props) => {
+  const {match} = props
+  const [currentQuest, setCurrentQuest] = useState<object>({})
+  // const id = [1, 2, 3]
+  return (
+    <section style={{border: "1px solid black",
+    height: '90vh',
+    marginTop: 'auto',
+    marginBottom: 'auto',
+    padding: '1em',
+    display: 'flex',
+    flexDirection: "column",
+    justifyContent: "center"}}>
+      <h2>Availabel Quests</h2>
```

Comment:
The spelling should be corrected to `Available`

Reply:
Done!